File: app/src/main/java/com/mdkashif/alarm/utils/AlarmApplication.java
package com.mdkashif.alarm.utils;

import android.app.Application;

import com.facebook.stetho.Stetho;
import com.github.omadahealth.lollipin.lib.managers.LockManager;
import com.mdkashif.alarm.R;
import com.mdkashif.alarm.security.AntiTheftUnlockActivity;

public class AlarmApplication extends Application {

    @SuppressWarnings("unchecked")
    @Override
    public void onCreate() {
        super.onCreate();

        // TODO: remove at production!
        Stetho.initializeWithDefaults(this);
        LockManager<AntiTheftUnlockActivity> lockManager = LockManager.getInstance();
        lockManager.enableAppLock(this, AntiTheftUnlockActivity.class);
        lockManager.getAppLock().setLogoId(R.mipmap.ic_launcher_foreground);
    }
}

File: app/src/main/java/com/mdkashif/universalarm/utils/AlarmApplication.kt
package com.mdkashif.universalarm.utils

import android.app.Application

import com.facebook.stetho.Stetho
import com.github.omadahealth.lollipin.lib.managers.LockManager
import com.mdkashif.universalarm.R
import com.mdkashif.universalarm.security.AntiTheftUnlockActivity

class AlarmApplication : Application() {

    override fun onCreate() {
        super.onCreate()

        // TODO: remove at production!
        Stetho.initializeWithDefaults(this)
        val lockManager = LockManager.getInstance()
        lockManager.enableAppLock(this, AntiTheftUnlockActivity::class.java)
        lockManager.appLock.logoId = R.mipmap.ic_launcher_foreground
    }
}
